src/main/io/gps.c

```diff
@@ -252,11 +254,50 @@ void gpsInit(serialConfig_t *initialSerialConfig, gpsConfig_t *initialGpsConfig)
 
 void gpsInitNmea(void)
 {
+#ifdef COLIBRI_RACE
+    uint32_t now;
+#endif
+
     switch(gpsData.state) {
         case GPS_INITIALIZING:
+#ifdef COLIBRI_RACE
```

Comment:
Yes, please remove GPS autobaud configurtation changes from this PR.  Make a new PR for the GPS related changes so we can discuss the intention of the code and implementation.